### ISSUE TITLE
Add "Evidence Summary" text box in GCI

### ISSUE THREAD

**Issue by wrightmw** [ClinGen/clincoded#1437]
Curators need a place to write a summary of the evidence they have evaluated. This free text field would appear on the Summary page and will be added as part of the GCI approval process.

This ticket is similar in implementation to #1330

**Comment by wrightmw** [comment]
We could adapt the bottom of the Summary Table as an interim measure, until the Review/Approval process is implemented, to make it like the VCI:

![screen shot 2017-08-23 at 5 03 00 pm](https://user-images.githubusercontent.com/15131169/29643631-27ae0b42-8825-11e7-825b-7663c77e0082.png)

**Comment by wrightmw** [comment]
@jimmyzhen @selinad

**Comment by selinad** [comment]
I like that idea @wrightmw

**Comment by selinad** [comment]
Needs review on small gcwg per additional suggestions by @ErinRiggs

**Comment by wrightmw** [comment]
Current mockup for ultra-lite version of Summary Table for R13: https://docs.google.com/presentation/d/1owi-f5kcrdPiTRyWXzBha31M0BS4ZsFUHz65-uQXB0g

**Comment by wrightmw** [comment]
@jimmyzhen In latest instance when I have a Limited Classification and I click Update, then for about one second it shows the Classification as Moderate. It then reverts back to the correct Limited Classification, so it does autocorrect but I thought I should bring it to your attention because this may confuse curators.

**Comment by jimmyzhen** [comment]
This change request has been implemented as part of the classification workflow (#1490).

![image](https://user-images.githubusercontent.com/6956310/31503642-13682fce-af25-11e7-8405-e2e65466e64b.png)

**Comment by selinad** [comment]
That makes sense @jimmyzhen -- I saw it live and it looks great.

**Comment by wrightmw** [comment]
Evidence Summary box present and correct. Thank @jimmyzhen

**Comment by selinad** [comment]
Agreed -- evidence summary box there and saves and displays correctly.  thx @jimmyzhen !
